src/main/scala/com/horizon/exchangeapi/OrgsRoutes.scala

```diff
@@ -451,8 +456,8 @@ trait OrgsRoutes extends JacksonSupport with AuthenticationSupport {
         description = "access denied"
       ),
       new responses.ApiResponse(
-        responseCode = "404",
-        description = "not found"
```

Comment:
See above.

Reply:
Not sure what you mean on this one. This is `POST /orgs/{orgid}`, for adding a new org. Doesn't make sense to have a '404 NOT FOUND' when creating a new org, because of course it doesn't exist yet